src/Chapter21/Listing21.10.CancelingParallelLoop.cs

```diff
@@ -35,52 +36,58 @@ public static async Task Main()
             CancellationTokenSource cts =
```

Comment:
CancellationTokenSource is IDisposable.
```suggestion
            using CancellationTokenSource cts = new CancellationTokenSource();
```